arangod/VocBase/Methods/Collections.h

```diff
@@ -85,7 +85,7 @@ struct Collections {
     arangodb::velocypack::Slice const& properties, // collection properties
     bool createWaitsForSyncReplication, // replication wait flag
     bool enforceReplicationFactor, // replication factor flag
-    FuncCallback const& callback // invoke on collection creation
+    FuncCallback callback // invoke on collection creation
```

Comment:
it's usually good practice to use 'const&' with lambdas to avoid making a copy of a lambda